README.md

```diff
@@ -67,7 +67,9 @@ Essa é a lista de moderadores e seus respectivos IDs do Slack:**
 
 * **David Conceição** - @davidtiagoconceicao
 * **Douglas Drumond** - @douglasdrumond
+* **JP Ventura** - @jpventura
 * **Marcello Galhardo** - @marcellogalhardo
+* **Rafa Araujo** - @orafaaraujo
```

Comment:
On Slack I'm @rafaelaraujo, with no letter O at the beginning

Reply:
@davidtcdeveloper username is correct. You also could markdown the github acounts

    * **David Conceição** - [@davidtiagoconceicao](https://github.com/davidtiagoconceicao)
    * **Douglas Drumond** - [@douglasdrumond](https://github.com/douglasdrumond)
    * **JP Ventura** - [@jpventura](https://github.com/jpventura)
    * **Marcello Galhardo** - [@marcellogalhardo](https://github.com/marcellogalhardo)
    * **Rafa Araujo** - [@rafaaraujo](https://github.com/rafaaraujo)
    * **Walmyr Carvalho** - [@walmyrcarvalho](https://github.com/walmyrcarvalho)